build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
     annotationProcessor 'org.projectlombok:lombok'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
-    implementation 'org.mifos:ph-ee-connector-common:1.9.1-SNAPSHOT'
+    implementation 'org.mifos:ph-ee-connector-common:1.10.0-SNAPSHOT'
```

Comment:
We are using `1.10.0` as connector common [PR](https://github.com/fynarfin/ph-ee-connector-common/pull/1) got merged and all the Airtel related DTOs are in `1.10.0`.